src/main/kotlin/com/geanik/cvgenerator/service/model/Template.kt

```diff
@@ -0,0 +1,7 @@
+package com.geanik.cvgenerator.service.model
+
+import com.geanik.cvgenerator.model.CvInformation
+
+interface Template {
+    fun generateFile(info: CvInformation) : ByteArray
```

Comment:
again not all templates return only 1 file
could be that a zip-file is returned, or only a pdf...
```suggestion
    fun populateTemplate(info: CvInformation) : ByteArray
```